runtimes/pythonrt/runner/main.py

```diff
@@ -20,9 +20,10 @@
 from grpc_reflection.v1alpha import reflection
 from syscalls import SysCalls
 import threading
-import os
 import time
 
+SERVER_GRACE = 3  # seconds
```

Comment:
Optional nit: maybe rename or add a comment that this is a server stop/shutdown grace period? "Server grace" is a little vague.